.github/workflows/Build_all.yml

```diff
@@ -145,17 +145,17 @@ jobs:
         source /opt/intel/oneapi/setvars.sh
         export FC=mpiifort
```

Comment:
also `rm -rf ./build`?

Reply:
For the first build of the step, no cleaning is needed